CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Changed
 
+- CSV config field `role-path` is now `role-paths` and takes a list of strings. Users can now specify multiple `Role` and `ClusterRole` manifests using `role-paths`. ([#1704](https://github.com/operator-framework/operator-sdk/pull/1704))
```

Comment:
Does this break `operator-sdk gen-csv ...` for existing CSV configs in projects that define a singular string field?
```YAML
role-path: "deploy/role.yaml"
```

If so then we should label this as a breaking change. Even if it is a small one.